src/main/scala/carts/cart/events/CartMoved.scala

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © 2022-2023 by Pervasive Cats S.r.l.s.
+ *
+ * All Rights Reserved.
+ */
+
+package io.github.pervasivecats
+package carts.cart.events
+
+import carts.cart.valueobjects.{CartId, Store}
+
+trait CartMoved {
+
+  val cartId: CartId
+  val store: Store
+
```

Comment:
Extra space at the end, missing spaces between vals, move to package "domainevents"